Source/WebKit/Shared/WebHitTestResultData.cpp

```diff
@@ -196,6 +200,19 @@ std::optional<WebKit::SharedMemory::Handle> WebHitTestResultData::getImageShared
     return imageHandle;
 }
 
+std::optional<FrameInfoData> WebHitTestResultData::frameInfoDataFromHitTestResult(const WebCore::HitTestResult& hitTestResult)
+{
+    auto* coreFrame = hitTestResult.frame();
+    if (!coreFrame)
+        return std::nullopt;
+
+    auto webFrame = WebFrame::fromCoreFrame(*coreFrame);
```

Comment:
`RefPtr webFrame = ...`

Reply:
I've changed all the places you noted to use explicit data types.